dockerfiles/PyClosureContainerDockerfile

```diff
@@ -3,12 +3,11 @@ FROM clipper/py-rpc:${CODE_VERSION}
 
 COPY clipper_admin/clipper_admin/python_container_conda_deps.txt /lib/
 RUN conda config --set ssl_verify no \
+  && conda install -y -c anaconda cloudpickle=0.5.2 \
   && conda install -y --file /lib/python_container_conda_deps.txt
-RUN conda install -c anaconda cloudpickle=0.5.2
 
 COPY containers/python/python_closure_container.py containers/python/python_closure_container_entry.sh /container/
 
-
-CMD ["/container/python_closure_container_entry.sh"]
+ENTRYPOINT ["/container/python_closure_container_entry.sh"]
```

Comment:
Changing this from `CMD` to `ENTRYPOINT` fixes the crash associated with the python call to `time.sleep`. I'll have to explore why this is the case. `CMD` executes the python script within `/bin/sh -c` directive, while `ENTRYPOINT` executes the python script in some other fashion; I'd imagine that this accounts for the difference.